Source/Scene/Model.js

```diff
@@ -800,6 +802,13 @@ define([
         options.cacheKey = cacheKey;
         var model = new Model(options);
 
+        if (!defined(options.headers)) {
+            options.headers = {};
+        }
+        if (!defined(options.headers.Accept)) {
```

Comment:
the call to `clone` above needs to be changed to pass `true` to deep clone, since we are modifying a sub-object now.